frontend/.eslintrc

```diff
@@ -70,7 +72,25 @@
     // It's OK to use ts-ignore when we're missing type definitions
     // for a JS library.
     "@typescript-eslint/ban-ts-ignore": "off",
-    "@typescript-eslint/ban-ts-comment": "off"
+    "@typescript-eslint/ban-ts-comment": "off",
+    // Permit for-of loops (https://stackoverflow.com/a/42237667)
+    "no-restricted-syntax": [
+      "error",
+      "ForInStatement",
+      "LabeledStatement",
+      "WithStatement"
+    ],
+    // Allow foo.hasOwnProperty("bar")
+    "no-prototype-builtins": "off",
+    // Disable a bunch of AirBNB rules we're currently in violation of.
+    // TODO: For each one, either fix and reenable, or provide a justification.
+    "import/extensions": "off",
+    "import/prefer-default-export": "off",
+    "import/no-extraneous-dependencies": "off",
```

Comment:
i think`import/no-extraneous-dependencies` should probably be left on. not sure why we would be importing something thats not in our package.json?

Reply:
The name is a bit misleading; the two issues that were triggered here are that
1. Some test-only code is imported as a devDependency which ESLint was suspicious of
2. Some of our third-party imports don't have correct eslint typings
I've gone ahead and just eslint-ignored those, and reenabled this rule.